lib/usb-hid-msc-lib.robot

```diff
@@ -33,3 +33,23 @@ Download ISO And Mount As USB
             Skip    unsupported
         END
     END
+
+Mount ISO As USB
+    [Documentation]    Mounts an ISO file from the local sources. In case of
+    ...    PiKVM we need to manually upload images to the device. For UEFI
```

Comment:
We don't, we can upload images via API as well, right? See: `Upload Image To PiKVM` kwd.

Reply:
Similar to https://github.com/Dasharo/open-source-firmware-validation/pull/187#discussion_r1601016146, I think we have only KWDs that needs url, so images generated with `scripts/secure-boot/generate-images/sb-img-wrapper.sh` would need to be

a) sent to cloud to have some link
b) hosted locally (?) so then we could start simple http server and upload from there

I think that we did not want to store those images anywhere (so the script to prepare images locally was created) and hosting them seems problematic? Wouldn't we need to know an ip address of the host?